manifest.json

```diff
@@ -448,16 +448,20 @@
                 "DeleteObjects",
                 "GetBucketCors",
                 "GetBucketEncryption",
+                "GetBucketTagging",
+                "GetBucketVersioning",
                 "GetObject",
                 "GetObjectAcl",
                 "HeadObject",
                 "ListBuckets",
                 "ListMultipartUploads",
+                "ListObjects",
```

Comment:
By curiosity, what is the use-case for this method? And why not using `ListObjectsV2`?

Don't forget to update the changelog BTW :wink: